lib/puppet/configurer/plugin_handler.rb

```diff
@@ -2,6 +2,7 @@
 # just included into the agent, but having it here makes it
 # easier to test.
 require 'puppet/configurer'
+INTERNATIONALIZED_AGENT_VERSION = Gem::Version.new("5.3.4")
```

Comment:
One nit, this defines a constant at the top-level. Better to move this inside of the `PluginHander` class definition so it's namespaced (to avoid naming collisions with arbitrary ruby code).